__tests__/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,9 @@
 const { humanCatDogYears } = require('../src');
 
 // Look Ma, no handlebars!!!
```

Comment:
You shouldn't leave comments in your git commits unless they are explaining a piece of code that would otherwise be hard to interpret.